src/users/store/effects/users.effects.ts

```diff
@@ -20,18 +21,25 @@ export class UsersEffects {
       ofType(usersActions.LOAD_USERS),
       switchMap((action: any) => {
         return this.usersService.getListOfUsers(action.payload).pipe(
-          map((userDetails) => {
+          concatMap((userDetails) => {
```

Comment:
What's the driver behind using concatMap here? has the getListOfUsers interface changed? Or is this resolving an issue

Reply:
Since I return an array of 2 actions, I had to switch to a concatMap. The compiler wouldn't let me through otherwise